e2e/cypress/integration/channel_sidebar/drag_and_drop_spec.js

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod @smoke
+// Group: @channel_sidebar
+
+import {testWithConfig} from '../../support/hooks';
+
+import {getRandomInt} from '../../utils';
+
+const SpaceKeyCode = 32;
+const DownArrowKeyCode = 40;
+
+describe('Channel sidebar', () => {
+    testWithConfig({
+        ServiceSettings: {
+            ExperimentalChannelSidebarOrganization: 'default_on',
+        },
+    });
+
+    before(() => {
+        cy.apiLogin('user-1');
+
+        cy.visit('/');
```

Comment:
Non-blocking and we can fix post-merge into master. Instead of using hard-coded user data like `user-1`, in the future please something like:
```javascript
cy.apiInitSetup({loginAfter: true}).then(({team}) => {
    cy.visit(`/${team.name}/channels/town-square`);
});
```

Reply:
I'll make a note of that. I forgot that we actually had any e2e tests for this since I filed separate tickets to add tests for this anyway.